public/js/notify.js

```diff
@@ -32,16 +32,26 @@ function setupNotification(callback) {
   }
 }
 
+const AUDIO = new Audio();
+// Maybe add the possibility for the notification to increase the sound's volume every n notifications?
```

Comment:
I'd keep comments as comments (explaining behavior / caveats of code) and instead turn this into a TODO

Reply:
Good call, I forgot to add the "TODO" part.